draft-ietf-quic-spin-exp.md

```diff
@@ -235,10 +235,17 @@ Implementations SHOULD allow administrators of clients and servers to disable
 the spin bit either globally or on a per-connection basis.
 Even when the spin bit is not disabled by the administrator implementations
 SHOULD disable the spin bit on a randomly chosen
-fraction of connections.  The selection process should be designed such that
-on average the spin bit is disabled for at least 1/8th of the connections.
+fraction of connections.
 
-When the spin bit is disabled, endpoints SHOULD set the spin bit value to zero,
+The selection process SHOULD be designed such that
+on average the spin bit is disabled for at least 1/8th of the connections, or
+1/8th of the paths when doing migrations. The random choice SHOULD be dependent
+on the source and destination addresses of the connection,
+so that the spin bit is consistently enabled or
+disabled for repeated connections between the same addresses.
+
+When the spin bit is disabled, endpoints SHOULD set the spin bit value to
+a constant value randomly chosen to be 0 or 1,
 regardless of the values received from their peer. Addendums or revisions to
```

Comment:
```suggestion
regardless of the values received from their peer.  Alternatively, endpoints MAY
change this value when changing connection ID.  Addendums or revisions to
```

Reply:
> disabled for repeated connections between the same addresses.

Isn't this the part that is hard to coordinate when multiple processes handle different connections - though you could share a hash key.

Reply:
Yes, this is why I added the extra clause about connection ID.  That should give enough wiggle room for that sort of implementation strategy (which is indistinguishable from having a load balancer).